index.html

```diff
@@ -110,6 +182,7 @@ <h2 class="tonic4">Multi-Post Stories</h2>
 					<li class="btn34">javaScript</li>
 				</ul>
 				<button class="btn44" type="submit">See Project</button>
```

Comment:
- [ ] Please implement the interactions for all the buttons. (enabled, pressed, etc).

![image](https://user-images.githubusercontent.com/83803414/186702703-5e2ce4db-dcb0-4d67-a536-27ad3dea0b7d.png)